.circleci/config.yml

```diff
@@ -171,6 +171,23 @@ jobs:
       environment: *environment
     steps: *steps
 
+  kafka-241:
+    working_directory: *working_directory
+    environment:
+      KAFKA_VERSION: "2.4.1"
+      KAFKA_SKIP_NETTEST: "1"
```

Comment:
Do you min explaining why these tests cannot pass with 2.4?

Reply:
Done. Also added a similar comment to where `KAFKA_SKIP_NETTEST` is read in the code.